pkg/driver_cryptocell/psa_cc/error_conversion.c

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (C) 2021 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_psa_crypto pkg_driver_cryptocell
+ * @{
+ *
+ * @file
+ * @brief       Glue code translating between PSA Crypto and the CryptoCell 310 driver APIs
+ *
+ * @author      Lena Boeckmann <lena.boeckmann@haw-hamburg.de>
+ *
+ * @}
+ */
+
+#include "psa_error.h"
+
+psa_status_t CRYS_to_psa_error(CRYSError_t error)
+{
+    switch(error) {
+        case CRYS_HASH_ILLEGAL_OPERATION_MODE_ERROR:
```

Comment:
This is using double indent. Please remove one level of indent in the `switch` body.